use boost::shared_ptr instead of std::shared_ptr for now

--- ur_kinematics/src/ur_moveit_plugin.cpp.orig	2018-01-06 14:52:56.000000000 +0100
+++ ur_kinematics/src/ur_moveit_plugin.cpp	2018-12-20 17:07:18.865848488 +0100
@@ -177,7 +177,7 @@
   rdf_loader::RDFLoader rdf_loader(robot_description_);
   const boost::shared_ptr<srdf::Model> &srdf = rdf_loader.getSRDF();
 #if ROS_VERSION_MINIMUM(1, 13, 0) 
-  const std::shared_ptr<urdf::ModelInterface>& urdf_model = rdf_loader.getURDF();
+  const boost::shared_ptr<urdf::ModelInterface>& urdf_model = rdf_loader.getURDF();
 #else
   const boost::shared_ptr<urdf::ModelInterface>& urdf_model = rdf_loader.getURDF();
 #endif
